utils/tf.py

```diff
@@ -67,40 +72,52 @@ def __create_config(self, intra_threads: int, inter_threads=1):
         :return: TensorFlow config
         """
         config = tf.compat.v1.ConfigProto()
-        config.allow_soft_placement = True
+        config.log_device_placement = False
         config.intra_op_parallelism_threads = intra_threads
         config.inter_op_parallelism_threads = inter_threads
         return config
 
-    def __initialize_graph(self, path_to_model: str):
+    def __initialize_graph(self, path_to_model: str, use_opsgraph=False):
         """
         A function initializing TF graph from frozen .pb model.
         :param path_to_model: str
         :return: TensorFlow graph
         """
-        graph = tf.compat.v1.Graph()
-        with graph.as_default():
+        if use_opsgraph:
+            graph = ops.Graph()
```

Comment:
Is this needed? Wouldn't tf.compat functions work with this?

Reply:
hmm, not sure, I tried many things none of which seemed to work. I will investigate it further.

Reply:
Seems like those two implementations differ however I haven't found many information easily about the differences online. 

https://stackoverflow.com/questions/53077164/difference-between-graph-pb2-graphdef-and-tf-graphdef